python/pyspark/sql/session.py

```diff
@@ -2010,11 +2010,19 @@ def addTag(self, tag: str) -> None:
         """
         Add a tag to be assigned to all the operations started by this thread in this session.
 
+        Often, a unit of execution in an application consists of multiple Spark executions.
+        Application programmers can use this method to group all those jobs together and give a
+        group tag. The application can use :meth:`SparkSession.interruptTag` to cancel all running
+        executions with this tag.
+
+        There may be multiple tags present at the same time, so different parts of application may
+        use different tags to perform cancellation at different levels of granularity.
+
```

Comment:
qq @HyukjinKwon : since this function is available only for Connect, but documented for regular session, shouldn't the doc say so? (same for other connect only functions)
Also, I wanted to add an example, but here examples will not work, because they would result in this exception (and pyspark testing actually checks if examples are runnable)

Reply:
They are actually documented as so in rst file, https://github.com/apache/spark/blob/master/python/docs/source/reference/pyspark.sql/spark_session.rst#spark-connect-only